continuous_integration/install.sh

```diff
@@ -72,18 +75,14 @@ create_new_conda_env() {
     # provided versions
     REQUIREMENTS=$(print_conda_requirements)
     echo "conda requirements string: $REQUIREMENTS"
-    conda create -n testenv --quiet --yes $REQUIREMENTS
+    conda create -n testenv python==$PYTHON_VERSION --yes
     source activate testenv
-    conda install pytest pytest-cov --yes
+    pip install $REQUIREMENTS
+    pip install pytest pytest-cov
 
     if [[ "$INSTALL_MKL" == "true" ]]; then
         # Make sure that MKL is used
-        conda install --quiet --yes mkl
-    elif [[ -z $CIRCLECI ]]; then
```

Comment:
why do we not need to uninstall mkl explicitly anymore?

Reply:
The `install.sh` is not used in CircleCI.